tools/geo/forestEstimationsImport.ts

```diff
@@ -20,21 +20,30 @@ export const forestIndicatorsImport = async (): Promise<void> => {
   // eslint-disable-next-line no-console
   const args = process.argv.slice(2)
 
-  const path = args[0]
-  const content = await fs.readFile(path)
+  const fileName = args[0]
+
+  const content = await fs.readFile(path.resolve(__dirname, fileName))
```

Comment:
can this just be 
```
  const content = await fs.readFile(path.resolve(__dirname, 'FRA....json'))
```
or there is a need to keep it dynamic ?

Reply:
Sure, we can omit the name, no need to keep it dynamic

Reply:
cool :D so anyone can just run the script, without worrying about file name to pass